### ISSUE TITLE
Node v10.7 - Trouble with module loader and importing objects after installation in node project

### ISSUE THREAD

**Issue by ganeshkbhat** [eivindfjeldstad/validate#75]
I am facing issues with module loader and importing objects after installation in node project.

I am trying to use this with Node V10.7.0 and am being thrown with errors saying the import statements are wrong (which is right since I am not behind a flag as well). Need steps to get this up and running cleanly with Node with or without babel and use it in a node project

**Comment by eivindfjeldstad** [comment]
Hmm, that's strange. The published version is compiled with Babel, so import statements shouldn't be an issue. Do you have a stack trace?

**Comment by ganeshkbhat** [comment]
It just doesn't work. Do I need a flag to run it? I am node 10.x

**Comment by eivindfjeldstad** [comment]
No, a flag should not be necessary. I assume you're installing with npm or yarn? It's very difficult to say what might be wrong without a stack trace though.
